### PR TITLE
User Agent Header Added

### PR DESCRIPTION
**Pull request by vaibhavbisht06**
This Pr will add user Agent Header to the request
For Example:
```
User-Agent: AmahiAnywhere/1.3 (iPad; iOS 7.0.4; Scale/2.00)
User-Agent: AmahiAnywhere/1.0 (Android 4.2.2)
```
#242 issue Resolved

### CODE REVIEW

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1030,15 +1030,15 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = AmahiAnywhere/AmahiAnywhere.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = BXD5262846;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1033]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1030,15 +1030,15 @@
...
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = AmahiAnywhere/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.vaibhav.AmahiAnywhere12345;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1041]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1054,15 +1054,15 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = AmahiAnywhere/AmahiAnywhere.entitlements;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = BXD5262846;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1057]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1054,15 +1054,15 @@
...
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = AmahiAnywhere/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.vaibhav.AmahiAnywhere12345;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:1065]: Please remove this change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ShresthPratapSingh) [reply AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj]: removed change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj (hunk 2 above)
Comment (ShresthPratapSingh) [reply AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj]: removed change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj (hunk 3 above)
Comment (ShresthPratapSingh) [reply AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj]: removed change

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj (hunk 4 above)
Comment (ShresthPratapSingh) [reply AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj]: removed change

**Comment by ShresthPratapSingh** [comment]
the conflicts in this branch are now resolved.

**Pull request merged**